tests/unit/test_routes/test_resource_retreival.py

```diff
@@ -0,0 +1,186 @@
+
+from configs import PaginatorConfig
+from datetime import datetime, timedelta
+from app.utils import get_error_code_from_status
+from app.versioning import LATEST_API_VERSION
+from yaml import load
+
+
+# TODO: We need negative unit tests (what happens when bad data is sent)
+def test_get_resources(module_client, module_db):
```

Comment:
We can probably break this into 3 different functions for the different test cases